src/style.css

```
@@ -1 +1,45 @@
 /* add styles here */
+html,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .star:nth-child(-n + 1) {
```

```suggestion
.stars--1 .star:nth-child(-n + 1),
.stars--2 .star:nth-child(-n + 2),
... {
```
Consider grouping selectors with commas